### PR TITLE
AgProduct & AgPackage schemas

### PR DESCRIPTION
**Pull request by brianwmunz**
Added AgProduct and AgPackage schemas.  Generally what this PR is attempting to do is reflect the needed information on a typical [case label](https://www.producetraceability.org/documents/FINAL_PTI_Harmonized_Traceabilty_Case_Label_April_2020_(003).pdf) for inspection.  Given that one case could contain multiple products, I've created an AgProduct schema to represent the products within the case, and AgPackage to represent the case itself.

In both the AgProduct and AgPackage schemas and their dummy data generation, we have a property to hold a hash representing the binary of an image.  This will be used in VC.  In order to do this I've pulled in the Open Food Trust image, converted to binary and used crypto-js to hash it into sha256.

### CODE REVIEW

File: docs/contexts/traceability-v1.jsonld [hunk 1]
```diff
@@ -14,10 +14,10 @@
       "@id": "https://w3id.org/traceability/AgInspectionReport",
       "@context": {
         "facility": {
-          "@id": "https://w3id.org/traceability/#facility"
+          "@id": "https://www.gs1.org/voc/Place"
         },
         "inspector": {
-          "@id": "https://w3id.org/traceability/#inspector"
+          "@id": "https://www.lawinsider.com/dictionary/food-inspector"
```
Comment (mprorock) [inline docs/contexts/traceability-v1.jsonld:20]: I think we need to break food inspector out to its own type if an appropriate one does not exist in GS1, else change reference to GS1

File: docs/schemas/AgPackage.json [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+            "description": "A four digit code digest of the GTIN code and Lot Code for use in integrating with voice picking systems.",
+            "type": "string"
+        },
+        "date": {
+            "$comment": "{\"term\": \"date\", \"@id\": \"https://schema.org/DateTime\"}",
+            "title": "Packing, Harvest, or Best By Date",
+            "description": "Human readable date describing the date the package was packed, harvested, or is 'best by.'",
+            "type": "string"
+        },
+        "labelImageUrl": {
```
Comment (mprorock) [inline docs/schemas/AgPackage.json:52]: URL in the name is likely correct here, but let's make sure we don't want that to be URI in the future.  I am open on this, but we will be living with whatever this naming convention is for a while

**Review by mprorock — CHANGES_REQUESTED** [review]
please double check usage of `@ id` - https://json-ld.org/spec/latest/json-ld/#node-identifiers by preference they will reference human readable page that has an underlying schema attached to it, but it is not clear to me from the spec if what is done here with links to non schema backed items is kosher or not.  @OR13 any thoughts as you have been working with json-ld longer than us?

**Comment by mprorock** [comment]
to clarify a bit on the request, we may want to specify `@ type` explicitly, as per https://json-ld.org/spec/latest/json-ld/#example-28-example-demonstrating-the-context-sensitivity-for-type

File: docs/contexts/traceability-v1.jsonld (hunk 1 above)
Comment (brianwmunz) [reply docs/contexts/traceability-v1.jsonld:20]: I feel like [here](https://github.com/w3c-ccg/traceability-vocab/issues/53#issuecomment-743262231) @OR13 is saying to point to something human readable for explaining the property which seems to imply by referencing wikipedia that not having a schema backing it is okay.  BUT that being said, the link above is not my favorite either.  In fact, finding definitions of inspector was not easy.  So if we decided to break inspector out to its own type would we revert back to what I did before where we represent the definition that we're creating?    I'm good either way.

**Comment by brianwmunz** [comment]
I feel like if we want to change the way the schemas generally use @type and @id we should make a separate PR because we'd have to change all of the existing schemas to the new format.  What do you think?

**Comment by mprorock** [comment]
I see your point.  I am going to say let's run with it this way for now, but if we run into issues at least we know the alternative path.  Will change PR to approved

**Review by mprorock — APPROVED** [review]

**Pull request merged**